packages/hs-react-ui/src/htmlElements/index.tsx

```diff
@@ -0,0 +1,21 @@
+import styled, { StyledComponentBase } from 'styled-components';
+import fonts from '../enums/fonts';
+
+const withGlobalStyle = (Component: string & StyledComponentBase<any, {}>) => styled(Component)`
+  box-sizing: border-box;
+  ${process.env.NODE_ENV !== 'test' ? fonts.importFonts : ''}
+  ${fonts.body}
+`;
+
+// Use these elements over native styled.xx elements, as they apply
+// sensible defaults for each element. If an element doesn't exist, add it to this block
+export const Div = withGlobalStyle(styled.div``);
+export const Span = withGlobalStyle(styled.span``);
+export const Button = withGlobalStyle(styled.button``);
+export const Input = withGlobalStyle(styled.input``);
+export const Label = withGlobalStyle(styled.label``);
+export const HR = withGlobalStyle(styled.hr``);
+export const Table = withGlobalStyle(styled.table``);
+export const TH = withGlobalStyle(styled.th``);
+export const TD = withGlobalStyle(styled.td``);
+export const TR = withGlobalStyle(styled.tr``);
```

Comment:
Can you add some info to the README that gives context about this change to onboarding developers so that they have a place they should be seeing to use these HOC's?

Reply:
Great point - I've made an update to the README.